src/synth/netlists-disp_vhdl.adb

```diff
@@ -623,16 +623,16 @@ package body Netlists.Disp_Vhdl is
             Disp_Template
               ("  \o0 <= std_logic_vector (resize (\ui0 * \ui1, \n0));" & NL,
                Inst, (0 => Get_Width (Get_Output (Inst, 0))));
-         when Id_Ult =>
+         when Id_Ult | Id_Slt =>
             Disp_Template ("  \o0 <= '1' when \ui0 < \ui1 else '0';" & NL,
                            Inst);
-         when Id_Ule =>
+         when Id_Ule | Id_Sle =>
             Disp_Template ("  \o0 <= '1' when \ui0 <= \ui1 else '0';" & NL,
                            Inst);
-         when Id_Ugt =>
+         when Id_Ugt | Id_Sgt =>
             Disp_Template ("  \o0 <= '1' when \ui0 > \ui1 else '0';" & NL,
                            Inst);
-         when Id_Uge =>
+         when Id_Uge | Id_Sge =>
             Disp_Template ("  \o0 <= '1' when \ui0 >= \ui1 else '0';" & NL,
                            Inst);
```

Comment:
I think you should handle Id_S* and Id_U differently otherwise the sign won't be considered.
Use `\si0` and `\si1` for signed conversions.